lib/dotcom/trip_plan/related_link.ex

```diff
@@ -104,10 +104,10 @@ defmodule Dotcom.TripPlan.RelatedLink do
     new("Massport schedules", url, :massport_shuttle)
   end
 
-  defp route_link(route, trip_id, itinerary) do
+  defp route_link(route, _, itinerary) do
     icon_name = Route.icon_atom(route)
     date = Date.to_iso8601(itinerary.start)
-    url = schedule_path(DotcomWeb.Endpoint, :show, route, date: date, trip: trip_id)
+    url = schedule_path(DotcomWeb.Endpoint, :show, route, date: date)
```

Comment:
This was just a legacy thing, as far as I can tell putting a trip in the query parameters doesn't render anything different on a schedule page nowadays...